kartograf/merge.py

```diff
@@ -1,20 +1,52 @@
 import ipaddress
 import shutil
-import numpy as np
-from pandarallel import pandarallel
 import pandas as pd
 
 from kartograf.timed import timed
 
+class Networks:
+    '''
+    A class whose _dict represents a mapping of the network number and IP networks within that network for a given AS file.
+
+    To check inclusion of a given IP network in the base AS file, we can compare (see check_inclusion) the networks under the root network number instead of all the networks in the base file.
+    '''
+    def __init__(self):
+        self._dict = dict()
+        self._keys = self._dict.keys()
+        for i in range(0, 255):
+            self._dict[i] = []
```

Comment:
the only reason to do this is to avoid having to look up if the key exists like at line 29 before updating the record, since we have to concat with the existing value (a list). Probably not a major performance impact, but avoids an extra lookup. It's cheap enough to do for IPv4 but would be more expensive for IPv6 since the range is larger. For IPv6 we check whether the key exists then set it.